### ISSUE TITLE
How Can I set Caffe CPU mode while installing the Ubuntu Pre-built package.

### ISSUE THREAD

**Issue by vuhailuyen1991** [NVIDIA/DIGITS#1182]
Hi there, are there any possible ways to set CPU_ONLY option to one if I use the .deb package?

**Comment by lukeyeager** [comment]
1. There's not any way to use the GPU build of Caffe if you don't have a GPU. Caffe's GPU build is sloppy and queries CUDA even when it doesn't need to.

2. But, you can still set DIGITS to CPU-only mode by setting `CUDA_VISIBLE_DEVICES=` (setting it to nothing).

**Comment by Bidski** [comment]
> But, you can still set DIGITS to CPU-only mode by setting CUDA_VISIBLE_DEVICES= (setting it to nothing).
@lukeyeager is this documented somewhere? I just ages trying to find this and only found results relating to DIGITS-2

**Comment by lukeyeager** [comment]
Even though we assume that most of our users do have a GPU, DIGITS still works if it doesn't detect any on your system.

Using `CUDA_VISIBLE_DEVICES` to trick your system into thinking there aren't any available GPUs is an unusual use case and not something we feel the need to document clearly. Besides, this envvar is a general CUDA feature - not specific to DIGITS.

P.S. Prior to https://github.com/NVIDIA/DIGITS/pull/1091 there was another way to select GPUs using a configuration file. But we chose to just follow standard practice now and use the `CUDA_VISIBLE_DEVICES` envvar.

**Comment by zahidmadeel** [comment]
Hello, 
I have just installed digits from [this link](https://github.com/NVIDIA/DIGITS/blob/master/docs/UbuntuInstall.md). I have downloaded the MNIST dataset as explained in getting started section but when I go on to build the model, it gives me error as asked in [this question](http://stackoverflow.com/questions/32320919/running-digits-in-cpu-mode-only). I have already built caffe with CPU_ONLY Flag set o 1 in MakeFile.config. Why is digits then showing me the  error. I have no nvidia GPU so I did not install any drivers or cuda. Is it required at all? How can I get around this issue?
Regards
Adeel
